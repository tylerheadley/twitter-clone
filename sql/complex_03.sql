/*
 * Calculates how commonly hashtag #coronavirus is used in each country.
 */
SELECT 
    country_code,
    count(*) as count
FROM tweet_tags
INNER JOIN tweets ON tweet_tags.id_tweets = tweets.id_tweets
WHERE
    tag = '#coronavirus'
GROUP BY country_code
ORDER BY count DESC,country_code;
