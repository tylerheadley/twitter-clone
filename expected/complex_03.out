 country_code | count 
--------------+-------
 ar           |     2
 es           |     2
 it           |     2
 au           |     1
 gb           |     1
 jp           |     1
 us           |     1
(7 rows)

